### PR TITLE
Skyline: fix a problem reading mz-only small molecule transition list…

### PR DESCRIPTION
**Pull request by bspratt**
…s with implied isotope labeling (that is, multiple precursor mz values for same charge) and unsorted precursor mz values. Skyline would become confused if first mz value was not smaller than all the others for the same molecule and charge.

### CODE REVIEW

File: pwiz_tools/Skyline/Model/SmallMoleculeTransitionListReader.cs [hunk 1]
```diff
@@ -117,6 +117,58 @@ public SrmDocument CreateTargets(SrmDocument document, IdentityPath to, out Iden
             var hasAnyMoleculeFormula = Rows.Any(row => !string.IsNullOrEmpty(GetCellTrimmed(row, INDEX_MOLECULE_FORMULA)));
             var hasAnyMoleculeCharge = Rows.Any(row => !string.IsNullOrEmpty(GetCellTrimmed(row, INDEX_PRECURSOR_CHARGE)));
             var hasAnyMoleculeAdduct = Rows.Any(row => !string.IsNullOrEmpty(GetCellTrimmed(row, INDEX_PRECURSOR_ADDUCT)));
+
```
[the remark this answers is not in the document]
Comment (bspratt) [reply pwiz_tools/Skyline/Model/SmallMoleculeTransitionListReader.cs]: Done.

**Comment by brendanx67** [comment]
@bspratt I thought you were going to try to get this into 19.2

**Comment by bspratt** [comment]
I thought we weren't, but I certainly can if that's what we want.

**Comment by brendanx67** [comment]
My memory was that only the Explicit RT Window fix was waiting until next release and this one just required you to refresh and fix conflicts, but was fine to commit. Let's get the conflicts resolved and get it into master. Then we can decide about cherry-picking back to skyline_19_2, even though that is the wrong direction which I just said we would try to avoid.

**Comment by bspratt** [comment]
If we want a clean cherry pick opportunity I should probably make a new branch for this. It's just two files changing but the tree is muddied now.

**Comment by bspratt** [comment]
committed the salient changes to 19.2 branch, and cherry picked that to master.

**Pull request closed without merging**